pkg/migration/azure.go

```diff
@@ -0,0 +1,83 @@
+package migration
```

Comment:
This is unrelated but do you know how such (business) logic is structured in upstream controllers?

Reply:
Nope.